frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,254 @@
+  namespace: openshift-storage
+spec:
+  serviceAccount:
+    metadata:
+      creationTimestamp: null
+  targetNamespaces:
+  - openshift-storage
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: ocs-catalogsource
+  namespace: openshift-marketplace
+spec:
+  displayName: Openshift Container Storage
+  icon:
+    base64data: ""
+    mediatype: ""
+  image: quay.io/ocs-dev/ocs-registry:latest
+  publisher: Red Hat
+  sourceType: grpc
+---`;
+
+describe('Testing OCS Subscription', () => {
+  beforeAll(async () => {
+    await execSync(`echo '${ocsDeploy}' | oc apply -f -`);
+    // Let the cluster add defns
+    await browser.sleep(45000);
+  }, 100000);
+
+  it('Subscribe to Operator', async () => {
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await goToOperatorHub();
+    await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    const ocs = await searchInOperatorHub('Openshift Container Storage');
+    ocs.click();
+    let uninstall = null;
+    try {
+      uninstall = !!(await uninstallOperator.isDisplayed());
+    } catch {
+      uninstall = false;
+    }
+    if (uninstall) {
+      // eslint-disable-next-line
+      console.log('Already subscribed to the operator');
+      await browser.wait(until.visibilityOf(closeOperatorInstall));
+      await closeOperatorInstall.click();
+      await operatorsPage();
+    } else {
+      await browser.wait(until.visibilityOf(installOperator));
+      await installOperator.click();
+      browser.refresh();
+      await installer.installOperator();
+    }
+    await browser.wait(until.visibilityOf(ocsOperator));
+    expect(ocsOperator.isDisplayed()).toBe(true);
+    await browser.wait(until.visibilityOf(ocsOperatorStatus));
+    let text = await ocsOperatorStatus.getText();
+    // If not installed yet wait for 15 seconds
+    if (!text.includes('Succeeded')) await browser.sleep(10000);
+    text = await ocsOperatorStatus.getText();
+    expect(text.includes('Succeeded') || text.includes('date')).toBe(true);
+  });
+
+  it(
+    'Create storage cluster',
+    async () => {
+      await operatorsPage();
+      await browser.wait(until.visibilityOf(ocsOperator));
+      await ocsOperator.click();
+      await browser.wait(until.visibilityOf(storageClusterLink));
+      await storageClusterLink.click();
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      // Node list fluctautes
+      await browser.sleep(3000);
+      await selectWorkerRows();
+      // Fluctating
+      await browser.sleep(1000);
+      await browser.wait(until.elementToBeClickable(btnCreate));
+      await btnCreate.click();
+      // Let it move to the other page
+      await browser.sleep(1000);
+      await browser.wait(until.visibilityOf(status));
+      await browser.wait(until.visibilityOf(crudView.resourceTitle));
+      const text = await crudView.resourceTitle.getText();
+      expect(text).toEqual('ocs-storagecluster');
```

Comment:
why? What are the possible values? Should I be testing for something else?

Reply:
As soon as we add day 2, things will get messy. So, it's better to do some damage control now.
Also, there are other scenarios which require similar data, all of it should come from same resource rather than everyone declaring multiple variables for same thing.

Reply:
I just checked the code. It's hardcoded to `ocs-storagecluster`. Can you elaborate on why we should move it to a separate initialization block? If the storage-cluster install code hard is hardcoded to the above name then what can cause problems here regarding the resource name?

Reply:
So you are telling me that, if later we decide to hardcode this to something else, we'll have to search through the entire repo for other occurrences of this and replace it, instead of just changing one variable? That is a good practice?